carbon-design-system/carbon-addons-iot-react
# fix: tvt issues with progress indicator, card editor forms and dashboard editor modals

**Summary**

- Fixed ProgressIndicator to use the correct fill for the current step
- Fixed ProgressIndicator to truncate long step titles
- Fixed all tooltips and icon descriptions in CardEditForm to use position: left to avoid being cut off on the right side
- Removed overflow: visible from DashboardEditor modals as it caused some the layout of the modal to break in different viewports

**Change List (commits, features, bugs, etc)**

- Updated snapshots

**Acceptance Test (how to verify the PR)**

1. Go to ProgressIndicator story and see that the first step with the long title is truncated correctly and the current step is filled in with blue (interactive-01)
2. Go to DashboardEditor, add a grouped bar chart, add a data item, hover over the icons next to the selected data items, and see the tooltip goes to the left
3. Go to DashboardEditor, add a table card, add a data item, hover over the icons next to the selected data items, and see the tooltip goes to the left
4. Go to DashboardEditor, add any card, hover over the data section info tooltip, and see the tooltip goes to the left
